src/cpip-pack

```diff
@@ -294,38 +308,42 @@ advanced_usage()
     echo     "$____"
 
     echo     "$____" "required arguments:"
-    echo     "$____" "  --name, -n PROJECT            Name of the project. Will be used for the name in"
-    echo     "$____" "                                the output $output."
+    echo     "$____" "  --name, -n PROJECT                  Name of the project. Will be used for the name in"
+    echo     "$____" "                                      the output $output."
     echo     "$____"
     echo     "$____" "optional arguments:"
-    echo     "$____" "  --file, -f FILE               Conda environment file. This option can be used"
-    echo     "$____" "                                multiple times to specify multiple files. The"
-    echo     "$____" "                                environment will be updated in the order that they"
-    echo     "$____" "                                are given on the command line."
-    echo     "$____" "  --poetry, -p DIR              Poetry project directory."
-    echo     "$____" "  --output, -o DIR              Directory where the final $output will go."
-    echo     "$____" "                                Otherwise, $output will be ouputed to the current"
-    echo     "$____" "                                working directory."
-    echo     "$____" "  --command, -c COMMAND         Command to run after the environment is setup."
-    echo     "$____" "                                This option can be used multiple times to specify"
-    echo     "$____" "                                multiple commands to run. They will be ran in the"
-    echo     "$____" "                                order that they are given on the command line."
+    echo     "$____" "  --file, -f FILE                     Conda environment file. This option can be used"
+    echo     "$____" "                                      multiple times to specify multiple files. The"
+    echo     "$____" "                                      environment will be updated in the order that they"
+    echo     "$____" "                                      are given on the command line."
+    echo     "$____" "  --poetry, -p DIR                    Poetry project directory."
+    echo     "$____" "  --output, -o DIR                    Directory where the final $output will go."
+    echo     "$____" "                                      Otherwise, $output will be ouputed to the current"
+    echo     "$____" "                                      working directory."
+    echo     "$____" "  --post-conda-command, -p COMMAND    Command to run after the conda environment is setup."
```

Comment:
`-p` is being used already and also is not linked to the command (see below function)

Reply:
thank you @john-higgins for the heads-up